docs/kubernetes/KubernetesPolicy.md

```diff
@@ -0,0 +1,62 @@
+# Programming Calico Policy in Kubernetes
+The [Calico-Kubernetes plugin](https://github.com/projectcalico/calico-docker/blob/master/docs/kubernetes/KubernetesIntegration.md) will allow you to program Networking policy directly from your Kubernetes pod config. 
+
+
```

Comment:
I think we want a pre-requisites section that links to "how to install the plugin" and also touches on the BenTheElder iptables-based kube-proxy requirement.